docs/news.rst

```diff
@@ -9,34 +9,31 @@ Release notes
 Python 3 Support (basic)
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-We have been hard at work to make Scrapy work on Python 3. Some features
-are still missing (and may never be ported to Python 3, see below),
-but you can now run spiders on Python 3.3, 3.4 and 3.5.
+We have been hard at work to make Scrapy run on Python 3. As a result, now you can run spiders on Python 3.3, 3.4 and 3.5, although some features are still missing (some of them may never be ported to Python 3).
```

Comment:
I'm in a 'fit text to 70-80 columns manually, don't rely on auto-wrap features in editors' camp :)